docs/technical/architecture.md

```diff
@@ -46,33 +46,12 @@ the `.jj/repo/store/type` file. There are similar files for the other backends
 
 ### Overview
 
-Here's a diagram showing some important types in the library crate. The
-following sections describe each component.
-
-```mermaid
-graph TD;
-    ReadonlyRepo-->Store;
-    ReadonlyRepo-->OpStore;
-    ReadonlyRepo-->OpHeadsStore;
-    ReadonlyRepo-->ReadonlyIndex
-    MutableIndex-->ReadonlyIndex;
-    Store-->Backend;
-    GitBackend-->Backend;
-    LocalBackend-->Backend;
-    LocalBackend-->StackedTable;
-    MutableRepo-->ReadonlyRepo;
-    MutableRepo-->MutableIndex;
-    Transaction-->MutableRepo;
-    WorkingCopy-->TreeState;
-    Workspace-->WorkingCopy;
-    Workspace-->RepoLoader;
-    RepoLoader-->Store;
-    RepoLoader-->OpStore;
-    RepoLoader-->OpHeadsStore;
-    RepoLoader-->ReadonlyRepo;
-    Git-->GitBackend;
-    GitBackend-->StackedTable;
-```
+Here's a diagram showing some important types in the library crate, and how they
+relate. For example, given a `Workspace`, you can use it to get a `WorkingCopy`
+or a `RepoLoader`. A `Transaction` is required to acquire a `MutableRepo`, etc.
+The following sections describe each component.
+
+![Type diagram](./types.svg)
```

Comment:
The drawback is of course that it's less maintainable. Can you add a link to the current version so the next person can fork it? I've never used Excalidraw but hopefully that's possible.

Reply:
Also no need for `./` in the link, to conform with the rest of our docs.